flight_planning/v1/flight_planning.yaml

```diff
@@ -135,25 +143,11 @@ components:
             - Closed
         uas_state:
           description: >-
-            State of the user's UAS associated with this flight plan.
-            
-              - `Nominal`: The user or UAS reports or implies that it is performing nominally, or has not indicated
-                `OffNominal` or `Contingent`.
-            
-              - `OffNominal`: The user or UAS reports or implies that it is temporarily not conforming to its intent,
-                but may expect to be able to recover to normal operation.
-            
-              - `Contingent`: The user or UAS reports or implies that it is not conforming to its intent and may be
-                unable to recover to normal operation.
-            
-              - `NotSpecified`: The UAS status is not currently available or known (for instance, if the flight is
-                planned in the future and the UAS that will be flying has not yet connected to the system).
-          type: string
-          enum:
-            - Nominal
-            - OffNominal
-            - Contingent
-            - NotSpecified
+            State of the user's UAS associated with this flight plan as reported by the operator.  If the UAS reports
```

Comment:
Should that field be removed from the `required` field then?
When telemetry is specified the UAS state will evolve over time as the telemetry is played by the USS.
If in the middle of that we update the flight plan, we probably want to leave that field empty.

In addition, some clarifications about what to do in such conflicting situations (i.e. state from flight plan vs state from telemetry) would probably be useful for implementers.

Reply:
Making it optional sounds reasonable.  I've also added (locally) clarification regarding potentially-conflicting values.